### PR TITLE
Add tcld namespace failover command

### PR DESCRIPTION
**Pull request by yux0**
## What was changed
Add tcld namespace failover command

## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

File: app/namespace.go [hunk 1]
```diff
@@ -391,6 +394,36 @@ func readAndParseCACerts(ctx *cli.Context) (read caCerts, err error) {
...
+	if len(namespace) == 0 {
+		return fmt.Errorf("namespace is required")
+	}
+
+	region := ctx.String(namespaceRegionFlagName)
+	if len(region) == 0 {
+		return fmt.Errorf("region is required")
+	}
+
+	cloudProvider := ctx.String(regionCloudProviderFlagName)
```
Comment (laniehei) [inline app/namespace.go:408]: we should probably just add in 'aws' as default until we're multicloud

File: app/namespace.go [hunk 2]
```diff
@@ -1208,6 +1241,49 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
...
+				},
+				&cli.StringFlag{
+					Name:  regionCloudProviderFlagName,
+					Usage: "The cloud provider of the region. Default: aws",
+					Value: "aws",
+				},
+			},
+			Action: func(ctx *cli.Context) error {
+				namespaceName := ctx.String(NamespaceFlagName)
+				region := ctx.String(namespaceRegionFlagName)
```
Comment (laniehei) [inline app/namespace.go:1270]: can we instead query their replicas and present them with a list of regions to fali over to?

**Review by laniehei — APPROVED** [review]

File: app/namespace.go (hunk 1 above)
Comment (yux0) [reply app/namespace.go:408]: This is optional

File: app/namespace.go (hunk 2 above)
Comment (yux0) [reply app/namespace.go:1270]: We can update the behavior in the future.

File: app/namespace.go [hunk 3]
```diff
@@ -391,6 +394,36 @@ func readAndParseCACerts(ctx *cli.Context) (read caCerts, err error) {
 	return parseCertificates(cert)
 }
 
+func (c *NamespaceClient) failoverNamespace(ctx *cli.Context) error {
+	namespace := ctx.String(NamespaceFlagName)
+	if len(namespace) == 0 {
+		return fmt.Errorf("namespace is required")
+	}
+
+	region := ctx.String(namespaceRegionFlagName)
```
[the remark this answers is not in the document]
Comment (yux0) [reply app/namespace.go:403]: for now, I want to make it explicit so customers know which region is active. I don't think it matters a lot even in a emergency case. They can look up where is the passive region.

**Pull request merged**